app/views/shared/_footer.html.haml

```diff
@@ -140,7 +140,7 @@
           = t '.footer_legal_text_html', {content_license: link_to('CC BY-SA 3.0', 'https://creativecommons.org/licenses/by-sa/3.0/'), code_license: link_to('AGPL 3', 'https://tldrlegal.com/license/gnu-affero-general-public-license-v3-(agpl-3.0)' )}
         %p.text-small
           %div
-            - cookies_policy_link = link_to( t( '.footer_data_cookies_policy' ), '', 'cookies-policy-modal' => true, 'cookies-banner' => !CookiesConsent.new(cookies, request.host).exists? && Spree::Config.cookies_consent_banner_toggle)
+            - cookies_policy_link = link_to( t( '.footer_data_cookies_policy' ), '', 'cookies-policy-modal' => true, 'cookies-banner' => !Web::CookiesConsent.new(cookies, request.host).exists? && Spree::Config.cookies_consent_banner_toggle)
```

Comment:
yes, I have moved this into an helper, the same helper where I require CookieConsent. I think it's much better now.
We could require Spree in the engine, for now, because the helper is for the footer that is in the main app, I'd leave the helper in the main app. I think it's fine. When we move the footer to the web engine, we can move the helper as well.